### DISCUSSION TITLE
end of support date details

### DISCUSSION THREAD

**Discussion by Aishbeck** [cake-build/cake#4347] [General]
I am looking for end of support date for the following components to feed in SBOM document
Microsoft.Diagnostics.Tracing.EventSource  - 1.1.28.0
Microsoft.Practices.ServiceLocation - 1.3.0
Microsoft.Practices.Unity.RegistrationByConvention - 4.0.1.0
Microsoft.Rest.ClientRuntime - 2.3.21.0
Open XML SDK - 2.17.1
System.Composition - 4.6.24705.01
System.IO.Packaging - 3.1.0
WindowsAzure.Storage - 9.3.2.0
Microsoft.Azure.AMQP - 2.4.7
Microsoft.Azure.Relay - 2.0.01559.6

**Comment by pascalberger** [comment]
This is the discussion space of the Cake Build orchestrator. You need to ask your question with the corresponding vendor.

**Reply by Aishbeck** [reply]
![Capture](https://github.com/user-attachments/assets/77c79584-286a-4cb2-a2d9-51ea2c0d45e6)

**Reply by Aishbeck** [reply]
it says to contact the maintainers for few packages..

**Reply by devlead** [reply]
We're not the maintainers of those packages.
